apps/src/templates/sectionProgressV2/InviteToV2ProgressModal.jsx

```diff
@@ -89,16 +90,11 @@ function InviteToV2ProgressModal({
     analyticsReporter.sendEvent(EVENTS.PROGRESS_V2_ACCEPT_INVITATION, {
       sectionId,
     });
-    setHasSeenProgressTableInviteData();
+    setHasSeenProgressTableInviteData(true);
     setHasSeenProgressTableInvite(true);
-    onShowProgressTableV2Change();
     setShowProgressTableV2(true);
-  }, [
-    sectionId,
-    setHasSeenProgressTableInvite,
-    setShowProgressTableV2,
-    onShowProgressTableV2Change,
-  ]);
+    new UserPreferences().setShowProgressTableV2(true);
```

Comment:
Why did we add this line? Won't this make a redundant post call?

Reply:
Sorry - I just updated the PR.  I added it because I thought we needed it after removing the `onShowProgressTableV2Change` but we don't.  It still feels like something is missing, but I as far as I can tell we copied over the needed behavior from `onShowProgressTableV2Change`.